WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsInterface.java

```diff
@@ -338,6 +339,12 @@ public boolean getShowRelatedPostImages() {
         return mSettings.showRelatedPostImages;
     }
 
+    public @NonNull String getRelatedPostsDescription() {
+        if (mActivity == null) return "";
+        String desc = mActivity.getString(getShowRelatedPosts() ? R.string.on : R.string.off);
```

Comment:
`getShowRelatedPosts() ` sounds weird to me. Should it be `isShowRelatedPosts()` ?